### DISCUSSION TITLE
API:Receipts - discount code/coupons, and Web bug

### DISCUSSION THREAD

**Discussion by TomasEm** [etsy/open-api#793] [Ideas]
Hi,
it will be great, if coupon name can be accessed from receipts/payments from API as (the second thing) on Dashboard, there is bug, that coupon of cancelled orders is counted in coupon stats.

Is there any change of this?

Tomas

**Comment by kross51** [comment]
I also see that the json payload for API call DOES NOT return "discount_code"; please add this datapoint.  This is all that I see in the json for discount:

...
"discount_amt": {
                "amount": ...,
                "divisor": ...,
                "currency_code": "USD"
            }
...
